### PR TITLE
Correcting documentation typo for loopTool code block 

### PR DESCRIPTION
**Pull request by hunterbarber**
## Fixes/Resolves:
Found some typos to recently added documentation

## Summary/Motivation:
Correcting documentation

## Changes proposed in this PR:
- Minor correction to typo

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/how_to_guides/how_to_use_loopTool_to_explore_flowsheets.rst [hunk 1]
```diff
@@ -232,26 +232,26 @@ Here we setup a simple run on RO erd flowsheet, requesting loopTool to run PS to
             - pressure_exchanger
             - pump_as_turbine
          sweep_param_loop
-            membrane_cost:  # Runs over differnt membrnae costs
+            membrane_cost:  # Runs over differnt membrnae costs, generating 3 steps
```
Comment (lbianchi-lbl) [inline docs/how_to_guides/how_to_use_loopTool_to_explore_flowsheets.rst]:
```suggestion
            membrane_cost:  # Runs over different membrane costs, generating 3 steps
```

File: docs/how_to_guides/how_to_use_loopTool_to_explore_flowsheets.rst [hunk 2]
```diff
@@ -232,26 +232,26 @@ Here we setup a simple run on RO erd flowsheet, requesting loopTool to run PS to
...
                type: LinearSample
                param: fs.costing.reverse_osmosis.membrane_cost
                lower_limit: 10
                upper_limit: 30
                num_samples: 3
-            factor_membrane_replacement:  # Runs over membrane_replacment costs, generating 10 steps
+            factor_membrane_replacement:  # Runs over membrane_replacment costs, generating 3 steps
```
Comment (lbianchi-lbl) [inline docs/how_to_guides/how_to_use_loopTool_to_explore_flowsheets.rst]:
```suggestion
            factor_membrane_replacement:  # Runs over membrane_replacement costs, generating 3 steps
```

**Review by lbianchi-lbl — COMMENTED** [review]
A couple of additional typos caught my eye.

**Comment by hunterbarber** [comment]
> A couple of additional typos caught my eye.

Thanks, I may as well do a larger review of this to see if I can find anymore before merging

**Review by adam-a-a — APPROVED** [review]
LGTM

**Review by agarciadiego — APPROVED** [review]

**Comment by lbianchi-lbl** [comment]
The in-progress checks are most likely stuck because of #1158. Since the other tests are passing (and the changes only affect the docs anyway) I'm merging this in.

image

**Pull request merged**